syntax = "proto3";

package infgen;

message report {
  uint32 client_id = 1;
  uint32 completes = 2;
  message notice {
    bool online = 1;
  }
  notice note = 3;
}

message command {
  uint32 conn = 1;
  uint64 start_ts = 2;
  uint32 duration = 3;
	uint32 length_mode = 4;
	uint32 length = 5;
	uint32 idt_mode = 6;
	uint32 lambda = 7;
  uint32 interact_times = 8;
	int32 think_time = 9;
}

